report/main.py

```diff
@@ -0,0 +1,476 @@
+                    -pdf-frame-content: header_content;
+                    left: 50pt; width: 512pt; top: 50; height: 350pt;
+                }}
+                @frame col1_frame {{             /* Content frame 1 */
+                    left: 44pt; width: 245pt; top: 200pt; height: 365pt;
+                }}
+                @frame col2_frame {{             /* Content frame 2 */
+                    left: 323pt; width: 245pt; top: 200pt; height: 365pt;
+                }}
+                @frame footer_frame {{           /* Static frame */
+                    -pdf-frame-content: footer_content;
+                    left: 50pt; width: 512pt; top: 650pt; height: 330pt;
+                }}
+            }}
+            body {{
+                font-family: Arial, sans-serif;
+                font-size: 18px;
+                text-align: left;
+            }}
+            h1 {{
+                background-color: {header_color};
+                color: {text_color};
+                padding-bottom: 25px;
+                padding-top: 50px;
+                font-size: 32px;
+                text-align: center;
+                }}            
+            h2 {{
+                background-color: {header_color};
+                color: {text_color};
+                padding-top: 10px;
+                font-size: 11px;
+                text-align: center;
+            }} 
+            .myDiv {{
+                border: 1px solid black; 
+                padding: 5px;
+                background-color: #f5f5f5;
+            }}   
+            .myDiv2 {{
+                text-align: center;
+            }}            
+            </style>
+    </head>
+
+    <body>
+        <div id="header_content">
+            <h1>New Release Report</h1>
+                <div class = "myDiv2">
+                    <p>Number of new releases: {new_releases}&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Top rated release: {top_rated_release}</p>
+                </div>
+            </div>
+        <div id="footer_content">
+            <h1>SteamPulse</h1>
+        SteamPulse - page <pdf:pagenumber>
+        </div>
+
+        <h2>Number of Reviews per Release</h2>
+        <img src="{fig1}" alt="Chart 1">
+
+        <h2>New Releases per Day</h2>
+        <img src="{fig2}" alt="Chart 2">
+        
+        <h2>New Reviews per Day</h2>
+        <img src="{fig4}" alt="Chart 3">
+
+        <h2>New Reviews per Day</h2>
+        <img src="{fig3}" alt="Chart 4">
+
+        <p>Trending games:</p>
+        <div class = "myDiv">
+            <p>{trending_game_one}</p>
+        </div>
+        <div class = "myDiv">
+            <p>{trending_game_two}</p>
+        </div>
+        <div class = "myDiv">
+            <p>{trending_game_three}</p>
+        </div>
+    </body>    
+    </html>
+    '''
+
+    with open("test.html", "w") as file:
+        file.write(template)
+
+    convert_html_to_pdf(template, environ.get("REPORT_FILE"))
+
+
+def send_email():
```

Comment:
I'm assuming typehints and docstrings aren't finished so ignore some of the above